### PR TITLE
[vtadmin-web] Remove placeholder links from NavRail

### PR DESCRIPTION
**Pull request by doeg**
## Description

- Removes the placeholder links in the NavRail, since they just 404. :) 

- Uses text instead of SVG icons for the single-character NavRail "hotkeys"; scare quotes since they aren't bound to any keyboard shortcuts (yet). I opted for this change as I moved "Workflows" under the other infrastructure links (i.e., those with counts) but the previous icon, a wrench, looked out of place... and exporting icons from Figma is sort of a pain. (Even if we switch to something like fontawesome for our icons, I think rendering these "icons" with CSS is still more lightweight + consistent than messing around with SVG!) 

- Deletes the `letter*.svg` files that we're no longer using. 

Before and after, for comparison. The new "icons" are noticeably lighter, so I'm definitely open to opinions. :) 

**Before:**
Screen Shot 2022-04-29 at 3 21 01 PM

**After:**
Screen Shot 2022-04-29 at 3 20 54 PM

## Related Issue(s)

N/A

## Checklist

-   [x] "Backport me!" label has been added if this change should be backported
-   [x] Tests were added or are not required
-   [x] Documentation was added or is not required

### CODE REVIEW

File: web/vtadmin/src/components/NavRail.module.scss [hunk 1]
```diff
@@ -117,3 +117,24 @@ a.navLink:focus .icon,
 a.navLink:hover .icon {
     fill: var(--colorPrimary);
 }
+
+.hotkey::before {
+    $hotkeySize: 16px;
+
+    @apply rounded-sm;
```
Comment (doeg) [inline web/vtadmin/src/components/NavRail.module.scss:124]: I never really know where to draw the line between `@apply` and just writing out CSS rules with variables, so open to suggestions here!

Comment (notfelineit) [reply web/vtadmin/src/components/NavRail.module.scss:124]: Me neither! I'd only say that since we have tailwind, I'd personally use it to its ✨ fullest ✨

**Review by notfelineit — APPROVED** [review]
I like the hotkey abstraction!!

**Comment by doeg** [comment]
The `Cluster (20)` test is failing (see https://github.com/vitessio/vitess/issues/10146) but since it's not actually required, I'm going to go ahead and merge this in. :)

**Pull request merged**